determiner.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python3
+
+import argparse
+import binascii
+import os
+import sys
+
+## print header first 
+## print header first
+authorSignature = '\nedeterminer.py - Published by Laura Tzigiannis and Moritz Nentwig\n'
+authorSignature += '---------------------------------------------------------------\n'
+print(authorSignature)
+
+## tests
+def check_file_extension(file): 
+    file_extension = file[file.rfind("."):]
```

Comment:
there are methods in python that search explicit for the file extension.
https://stackoverflow.com/questions/541390/extracting-extension-from-filename-in-python

Reply:
Done